mitodl/open-discussions
# Display link post thumbnails in list view

#### What are the relevant tickets?
Closes #778 

#### What's this PR do?
Displays a thumbnail at the top right of each link post card.  The thumbnail is cropped and sized via the Embedly API (103 x 201 pixels as in [Zeplin design](https://app.zeplin.io/project/5b181d3bf71408ae4d406b94)).  CSS is used to crop further to 103x94 pixels in mobile/phone mode.

#### How should this be manually tested?
Create a mix of text and list posts in a channel and then view the channel's posts list.  Each link post should display a thumbnail, 103x201 pixels.  If you reduce the width of your browser window enough, the thumbnail should shrink to 103x94 pixels.  Clicking on the thumbnail should open the link in a new tab.

#### Any background context you want to provide?
https://app.zeplin.io/project/5b181d3bf71408ae4d406b94

Wide browser:
<img width="714" alt="screen shot 2018-07-23 at 2 08 38 pm" src="https://user-images.githubusercontent.com/187676/43094828-fe921eb0-8e81-11e8-9180-67b540bfe81b.png">

Narrow browser/mobile:
<img width="469" alt="screen shot 2018-07-23 at 2 08 50 pm" src="https://user-images.githubusercontent.com/187676/43094826-fe80f96e-8e81-11e8-9076-9e7fcb77d74d.png">
